mlkem/native/aarch64/aarch64_zetas.c

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2024 The mlkem-native project authors
+// SPDX-License-Identifier: Apache-2.0
+
+// WARNING: This file is auto-generated from scripts/autogenerate_files.py
```

Comment:
Of this file is auto-generated, why both to add it to the repo at all? Why not just re-generate it on demand as part of the build process? I guess that creates a dependence on Python?

Reply:
To make it easy to use